src/shared/utils/exportData/getReportCSVObject.test.ts

```diff
@@ -160,8 +160,63 @@ const result = {
   legacy_user_id: '',
 };
 
+const newResult = {
```

Comment:
A bit of a nit, but let's just call this `result` as it is the result when the flag is on and also when it is ultimately archived. Minimizes cleanup when flag is removed.